Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -705,8 +709,65 @@ public static void HandleListPlayers(Session session, params string[] parameters
             {
                 var listPlayersMessage = new GameMessageSystemChat(message, ChatMessageType.Broadcast);
                 session.Network.EnqueueSend(listPlayersMessage);
-            } else
+            }
+            else
                 Console.WriteLine(message);
         }
+
+        // This debug command was added to test combat stance - we need one of each type weapon and a shield Og II
+        [CommandHandler("weapons", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0,
+            "Creates 1 of each weapon class in your inventory.")]
+        public static void HandleWeapons(Session session, params string[] parameters)
+        {
+            HashSet<uint> weaponsTest = new HashSet<uint>() { 93, 148, 300, 307, 311, 326, 338, 348, 350, 12748, 12463, 31812 };
+            ActionChain chain = new ActionChain();
+
+            chain.AddAction(session.Player, () =>
+            {
+                foreach (uint weenieId in weaponsTest)
+                {
+                    WorldObject loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
```

Comment:
I recognize this function is not introduced by your PR, so I wont reject, but...

I'm not a big fan of the LootGenerationFactory modifying the player in the player's context -- logically the Player should modify itself in its own context. IMO: The job of a LootGenerationFactory is to generate loot, not modify players.

This interface is just asking to be abused.  By accepting a player argument it hints that it will safely modifies the player, which it does not.